kbs/docs/initdata.md

```diff
@@ -0,0 +1,237 @@
+# Initdata Specification
```

Comment:
Thanks @Xynnn007 !

I suggest breaking this document into 2 separate sections or separate documents:
a. Generic Init Data definition.
b. How CoCo will use Init Data in the near future.

For example, section (b) should say that CoCo will support initially just TOML format for Init Data (no JSON or YAML).

Reply:
Great. Let me bring back the use case section with only overall introductions.

Reply:
Same thread as https://github.com/confidential-containers/trustee/pull/348#discussion_r1520793686